openapi/openapi.yaml

```
@@ -1,4 +1,4 @@
-openapi: "3.0.0"
+openapi: "3.1.2"
```

```suggestion
openapi: "3.0.0"
```
enum is available in openapi 3.0, so let's keep that.